src/App.tsx

```diff
@@ -1,10 +1,56 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+
+import { Filter } from './Components/Filter';
+import { Header } from './Components/Header';
+import { ErrorMessage } from './Components/ErrorMessage';
+import { TodoList } from './Components/TodoList';
+import { FilterOptions } from './types/FilterOptions';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 10963;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [filter, setFilter] = useState<FilterOptions>(FilterOptions.ALL);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((todosFromServer: Todo[]) => setTodos(todosFromServer))
```

Comment:
```suggestion
      .then(setTodos)
```
Add return type for getTodos